### PR TITLE
[PLAY-551] Testing React-Leaflet with Leaflet CDN

### PR DESCRIPTION
**Pull request by nidaqg**
DRAFT PR FOR INVESTIGATION

#### Screens
Screenshot 2023-01-19 at 1 19 41 PM

## [Gist on issues faced and solutions applied](https://gist.github.com/nidaqg/8342698962bd17359c27e255c0e6f87f)

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_map/_map.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import "../tokens/typography";
+@import "../tokens/colors";
+//must bring leaflet css in via cdn to avoid webpack parsing issues
+@import url("https://unpkg.com/leaflet@1.9.3/dist/leaflet.css");
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_map/_map.scss:4]: I know we already discussed, but just reminder to move this to site-style.css so it doesn't get compiled in our dist.

Comment (nidaqg) [reply playbook/app/pb_kits/playbook/pb_map/_map.scss]: done!

File: playbook-website/app/javascript/site_styles/_site-style.scss [hunk 2]
```diff
@@ -4,6 +4,9 @@
 @import "playbook-ui/dist/tokens/border_radius";
 @import "playbook-ui/dist/tokens/shadows";
 
+//must bring leaflet css in via cdn to avoid webpack parsing issues
+@import url("https://unpkg.com/leaflet@1.9.3/dist/leaflet.css");
```
Comment (nidaqg) [inline playbook-website/app/javascript/site_styles/_site-style.scss:8]: Leaflet CSS does not play well with webpack 4. Bringing the CSS in through the dist throws several parsing errors. Bringing it in via CDN fixes all those issues and is also the methods of consumption recommended by Leaflet itself.

File: playbook-website/package.json [hunk 3]
```diff
@@ -9,9 +9,12 @@
   },
   "dependencies": {
     "@rails/webpacker": "5.2.1",
+    "leaflet": "1.7.1",
     "playbook-ui": ">= 1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-leaflet": "3.1.0",
```
Comment (nidaqg) [inline playbook-website/package.json:16]: Anything above 3.1.0 breaks the map because of JS parsing errors coming from react-leaflet

**Comment by nidaqg** [comment]
Closing this PR as we are no longer using react-leaflet for the Map kit.

**Pull request closed without merging**